json-rpc/docs/README.md

```diff
@@ -0,0 +1,43 @@
+
+# JSON-RPC SPEC
+
+## Overview
+
+The Libra client API is based on the JSON-RPC protocol. This specification defines the client API endpoints and types, and provides usage examples.
+
+List of released stable methods:
+
+* [submit](method_submit.md)(data: string) -> void
+* [get_transactions](method_get_transactions.md)(start_version: unsigned_int64, limit: unsigned_int64, include_events: boolean) -> List<[Transaction](type_transaction.md)>
+* [get_account](method_get_account.md)(account: string) -> [Account](type_account.md)
+* [get_account_transaction](method_get_account_transaction.md)(account: string, sequence_number: unsigned_int64, include_events: boolean) -> List<[Transaction](type_transaction.md)>
+* [get_account_transactions](method_get_account_transactions.md)(account: string, start: unsigned_int64, limit: unsigned_int64, include_events: boolean) -> [Transaction](type_transaction.md)
+* [get_metadata](method_get_metadata.md)(version: unsigned_int64) -> [Metadata](method_get_metadata.md#type-metadata)
+* [get_events](method_get_events.md)(key: string, start: unsigned_int64, limit: unsigned_int64) -> List<[Event](type_event.md)>
+* [get_currencies](method_get_currencies.md)() -> List<[CurrencyInfo](method_get_currencies.md#type-currencyinfo)>
+
+Note: unless specifically mentioned, all parameters are required for the method.
+
+## JSON-RPC specification
+
+JSON-RPC is a stateless, light-weight remote procedure call (RPC) protocol. Refer to the [JSON-RPC Specification](https://www.jsonrpc.org/specification) for further details.
+
+
+### Batched requests
+
+The JSON-RPC protocol allows requests to be batched. An arbitrary number of requests (max 20 by default) can be combined into a single batch and submitted to the server. These requests will be processed together under a single request context.
+
+
```

Comment:
we should add an section

### Libra extensions
libra_chain_id
libra_ledger_version
libra_ledger_timestampusec

I can contribute more if needed